lib/OpenQA/Scheduler/Client.pm

```diff
@@ -22,6 +22,17 @@ use OpenQA::Utils 'service_port';
 has client => sub { OpenQA::Client->new(api => 'localhost') };
 has port   => sub { service_port('scheduler') };
 
+my $IS_SCHEDULER_ITSELF;
+sub mark_current_process_as_scheduler { $IS_SCHEDULER_ITSELF = 1; }
+sub is_current_process_the_scheduler { return $IS_SCHEDULER_ITSELF; }
```

Comment:
`current_process_is_scheduler`?

Reply:
I actually prefer `is_current_process_the_scheduler`. Maybe because boolean properties always start with `is/are` in Qt and I'm used to it.

Reply:
I feel like `current_process_is_scheduler` sounds more natural. Also, this could be setter and accessor if we consider it a trivial property with no side effects.

Reply:
`current_process_is_scheduler` sounds like an assertion and `is_current_process_the_scheduler´ sounds like a question. The purpose of this function is answering the question and not establishing or ensuring an assertion.